src/binaryen-c.h

```diff
@@ -262,7 +266,7 @@ BinaryenOp BinaryenHasFeature(void);
 typedef void* BinaryenExpressionRef;
 
 // Block: name can be NULL
```

Comment:
please also add another comment about the use of BinaryenUndefined for the last value, and what that means